src/include/Graphics/MipiDisplay.h

```diff
@@ -310,19 +145,19 @@ class Base : public SpiDisplay
 	 *
 	 * For example, if the display is flipped horizontally, use:
 	 * 
-	 * 		setDefaultAddressMode(Graphics::Display::Mipi::DCS_ADDRESS_MODE_MIRROR_X);
+	 * 		setDefaultAddressMode(Graphics::Display::Mipi::Mipi::DCS_ADDRESS_MODE_MIRROR_X);
```

Comment:
The comments here `Graphics::Display::Mipi::Mipi::DCS_....` should be `Graphics::Display::Mipi::DCS_...`